src/device/device_cold.cpp

```diff
@@ -0,0 +1,5 @@
+//
+// Created by Dusan Klinec on 10/08/2018.
+//
+
+#include "device_cold.hpp"
```

Comment:
What does this file do ?
Also new source files should have the Monero copyright notice (if you're not releasing your modifications under the exiting licence, the code isn't going in). You can keep your name and creation date in after it of course.